### PR TITLE
Minor formatting on doc examples

### PR DESCRIPTION
**Pull request by jrobinAV**
Replaced unused variable with _ in loops.

### CODE REVIEW

**Review by FredLL-Avaiga — APPROVED** [review]
does this break the linter ?

**Comment by jrobinAV** [comment]
> does this break the linter ?

nop

File: doc/gui/examples/charts/error-bars-asymmetric.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 data = {
     # The x series is made of random numbers between 1 and 10
-    "x": [random.uniform(1, 10) for i in y],
+    "x": [random.uniform(1, 10) for _ in y],
```
Comment (florian-vuillemot) [inline doc/gui/examples/charts/error-bars-asymmetric.py:27]: I prefer ;)
```suggestion
    "x": list(random.uniform(1, 10)),
```

**Review by florian-vuillemot — APPROVED** [review]

**Review by FabienLelaquais — APPROVED** [review]
👌

**Pull request merged**